src/test/scala/nl/knaw/dans/easy/stage/package.scala

```diff
@@ -29,6 +29,8 @@ package object stage {
     urls.map { url =>
       new URL(url).openConnection match {
         case connection: HttpURLConnection =>
+          connection.setConnectTimeout(1000)
```

Comment:
Decided against using [InetAddress](http://docs.oracle.com/javase/8/docs/api/java/net/InetAddress.html) because it would be necessary to first get the host from the URL.